.github/workflows/tests.yml

```diff
@@ -21,14 +21,6 @@ jobs:
           GITHUB_OAUTH_TOKEN: ${{ secrets.BUILD_GITHUB_OAUTH_TOKEN }}
         run: docker-compose build
 
-      - name: Prepare stubs
-        env:
-          AWS_ACCESS_KEY_ID: ${{ secrets.BUILD_AWS_ACCESS_KEY_ID }}
-          AWS_SECRET_ACCESS_KEY: ${{ secrets.BUILD_AWS_SECRET_ACCESS_KEY }}
-          AWS_S3_BUCKET: ${{ secrets.S3_BUCKET }}
-          AWS_REGION: ${{ secrets.S3_AWS_REGION }}
-        run: docker-compose run --rm production composer loadS3
-
```

Comment:
To nemožeš odebrat, teď prošly testy enom proto, že tam sů soubory od minula, ale když přidám nějaký nový stub tak to failne protože už sa nenahraje do S3. Je to vlastně to co sem psal u toho PR v common, že sa ten stub loader bude muset upravit aby tam šly nahrát ty soubory přes parametr, enom mě nedošlo že to bude potřeba už teď :( . Možná by teď stačilo pustit ten loadS3 z toho common.

Reply:
No jo vlastne 🤦 uz to chapem, tak si vravim, ze teda bud to odtialto vyhodit a nechat to zatial len v tej teradata libke, alebo to presunut niekam do zlozky mimo testy napr do `Share` kde by som to potom mohol pouzit v dalsich libkach

Reply:
Zas na druhu stranku je to len cisto na testu a v tom shared to teda asi nedava zmysel, tak som to zatial presunul sem

Reply:
Tak shared by měl byt aj základ testů. Když uděláš další driver třeba pro SNFLK tak tam budeš muset ten stub loader zas copy pastnůt (on je teda teď copy paste v import export aj tady :D). Ale klidně ho tu teď nech a pak stým něco vymyslíme, ono to není potřeba enom v driveru.